Tests/UnitTests/Mocks/MockOperationDispatcher.swift

```diff
@@ -12,7 +12,7 @@ class MockOperationDispatcher: OperationDispatcher {
     var invokedDispatchOnMainThread = false
     var invokedDispatchOnMainThreadCount = 0
     var shouldInvokeDispatchOnMainThreadBlock = true
-    var forwardToOriginalDispatchOnMainThread = false
+    var forwardToOriginalDispatchOnMainThread = true
```

Comment:
Changed to the MockOperationDispatcher would actually dispatch to the main thread in tests.

Will revert back.